packages/embeds/embed-core/playwright/config/playwright.config.ts

```diff
@@ -79,7 +80,8 @@ declare global {
     interface Matchers<R> {
       toBeEmbedCalLink(
         calNamespace: string,
-        getActionFiredDetails: Function,
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Don't love the idea of adding more `eslint-disable` rules but sometimes in these very generic cases (like in test suites) it's necessary. I see we do disable this in some other places where the code is generic. Open to suggestions here.

The change from not using `Function` was due to an eslint error.